movie.sh

```diff
@@ -1,58 +1,68 @@
 #!bin/bash
 
-function add_style()
-{
+mkdir tmp #Making a folder to dump files
+readonly NUM_ARGUMENTS=$#
+readonly DIRECTORY_NAME=$1
+
+function add_style(){
         tput bold
```

Comment:
too much indentation. Bash style guide suggests you should have only 2. I say you can do 4 as well if you like. But then it has to be consistent throughout the script. You can't have different indentation in the entire script.